onnxruntime/contrib_ops/cpu/quantization/matmul_nbits.cc

```diff
@@ -23,27 +24,137 @@
         nbits_{narrow<size_t>(info.GetAttr<int64_t>("bits"))} {
     ORT_ENFORCE(nbits_ == 4,
                 "Only 4b quantization is supported for MatMulNBits op, additional bits support is planned.");
+    info.GetAttrOrDefault<int64_t>("accuracy_level", &accuracy_level_, 0);
+    is_asym_ = info.GetInputCount() >= 4;
```

Comment:
It will be better to add a check if B, Scale, zero_point(if exist) are all constant values with API:
[https://github.com/microsoft/onnxruntime/blob/3588fbac1377eb2a74fcf82f8d8768c7c00397d3/include/onnxruntime/core/framework/op_kernel_info.h#L47](https://github.com/microsoft/onnxruntime/blob/3588fbac1377eb2a74fcf82f8d8768c7c00397d3/include/onnxruntime/core/framework/op_kernel_info.h#L47). It can avoid the weird case that B is constant but Scale is not but you still pack the value.

Reply:
fixed in 4c07742421372e23292c0cafd514d8df580fd608